### PR TITLE
remove the merge tool

### LINKED ISSUE #8875: Remove Merge Tool UI
Extracted from https://github.com/desktop/desktop/pull/8813#issuecomment-570256202

> the merge tool selection is behind a dev feature flag because we never ended up shipping it. At some point it might be a good idea to unship all of #2802 since it's been superseded by the in-app conflict resolution work.

### PR DESCRIPTION
**Pull request by Rexogamer**
Closes #8875

## Description

- Removes all traces of the merge tool.

## Release notes

Notes: no-notes, since it never shipped outside dev

### CODE REVIEW

**Comment by Rexogamer** [comment]
not entirely sure about that error

**Comment by shawnfunke** [comment]
I don't have the code checked out but it looks like in the `onSave` method you have one extra curly bracket at line 392 that shouldn't be there.
https://github.com/desktop/desktop/blob/6b37c9fc3b4239c38e24a7cb3cfda9fb7805dd80/app/src/ui/preferences/preferences.tsx#L363-L395

**Comment by outofambit** [comment]
@Rexogamer it looks like the build failed due to linting. maybe try another `yarn lint:fix`, commit, and push?

**Comment by Rexogamer** [comment]
I can’t really do that, sorry, but if anyone would be able to tell me what’s wrong I’ll fix it

**Comment by Rexogamer** [comment]
Actually I’m not sure if I want to remove the shell as it seems to be used for other things

File: app/src/ui/preferences/integrations.tsx [hunk 1]
```diff
@@ -144,43 +136,12 @@ export class Integrations extends React.Component<
...
-  }
-
   public render() {
     return (
       <DialogContent>
         <h2>Applications</h2>
         <Row>{this.renderExternalEditor()}</Row>
-        <Row>{this.renderSelectedShell()}</Row>
-        {this.renderMergeTool()}
+        <Row>{this.onSelectedShellChanged()}</Row>
```
Comment (niik) [inline app/src/ui/preferences/integrations.tsx]: Shouldn't this be `renderSelectedShell`?

```suggestion
        <Row>{this.renderSelectedShell()}</Row>
```

**Comment by pljhu9** [comment]
7b73796d47cdfb1300bf326af6755dd3a20356c2

**Pull request merged**